documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -186,6 +186,22 @@ By default `adapter-node` gracefully shuts down the HTTP server when a `SIGTERM`
 
 > If you want to customize this behaviour you can use a [custom server](#custom-server).
 
+You can listen to the `sveltekit:shutdown` event which is emitted after the HTTP server has finished shutting down. Unlike Node's `exit` event, the `sveltekit:shutdown` event is always emitted when a shutdown is triggered and also supports handling asynchronous code.
+
+```js
+process.on('sveltekit:shutdown', async (reason) => {
+  await db.close();
+  await jobs.stop();
+  process.exit();
```

Comment:
I don't think I'd recommend this. If you close your database, etc. then there should be no more work happening and the server should close by itself. It's kind of a cludge because there's something not being shutdown correctly
```suggestion
```